server/database/schemas/User.js

```diff
@@ -1,28 +1,40 @@
-const mongoose = require('mongoose');
-const { MongooseAutoIncrementID } = require('mongoose-auto-increment-reworked');
-const bcrypt = require('bcryptjs');
-const R = require('ramda');
+const mongoose = require("mongoose");
+const { MongooseAutoIncrementID } = require("mongoose-auto-increment-reworked");
+const bcrypt = require("bcryptjs");
+const R = require("ramda");
 
 const { Schema } = mongoose;
 
-const userSchema = new Schema({
-  username: { type: String, lowercase: true, required: true, unique: true, immutable: true },
-  username_case: { type: String, required: true },
-  password: { type: String, required: true },
-  profile_pic: { type: String },
-  first_name: { type: String, maxlength: 20 },
-  last_name: { type: String, maxlength: 20 },
-  bio: { type: String, maxlength: 240 },
-  created_at: { type: Date, default: Date.now, immutable: true },
-  updated_at: { type: Date },
-}, { versionKey: false });
-
-if (process.env.NODE_ENV !== 'test') {
-  MongooseAutoIncrementID.initialise('counters');
+const userSchema = new Schema(
+  {
```

Comment:
where is the city field?

Reply:
supposed to be in line 19. 
don't you see that?

Reply:
Now that you added it, I see :-)